docker-init.sh

```diff
@@ -3,11 +3,6 @@ echo "prepare environment"
 # replace {% REMARK_URL %} by content of REMARK_URL variable
 find . -regex '.*\.\(html\|js\|mjs\)$' -print -exec sed -i "s|{% REMARK_URL %}|${REMARK_URL}|g" {} \;
 
```

Comment:
what was the  reason to remove this replacement with sed?

Reply:
Because I didn't add the `web` directory to the Dockerfile. But I decided to keep it there for simplicity.